aten/src/ATen/SparseTensorUtils.h

```diff
@@ -17,7 +17,7 @@ using SparseType = Type;
 //
 // This may be called repeatedly, so make sure it's pretty cheap.
 inline SparseTensorImpl* get_sparse_impl(const SparseTensor& self) {
-  AT_ASSERTM(!self.is_variable(), "_internal_get_SparseTensorImpl: should not be a variable");
+  AT_ASSERTM(!self.is_variable(), "_internal_get_SparseTensorImpl: should not be a variable");  // TODO: remove this when Variable and Tensor are merged
```

Comment:
Currently we need this check because `self.unsafeGetTensorImpl()` doesn't return the underlying data tensor's TensorImpl when this function is called on a Variable. After Variable and Tensor are merged, `self.unsafeGetTensorImpl()` will return the right TensorImpl and hence we don't need this check anymore.